dist/json-formatter.css

```diff
@@ -0,0 +1,194 @@
+.json-formatter-row {
```

Comment:
do not commit this

Reply:
Do you mean the entire dist folder? I appeared because I made sub sequential changes without creating  a different branch. I will recreate the PR without it.